docs/reference/src/documentation/language/style-guide/unused-variables.md

```diff
@@ -0,0 +1,13 @@
+# Unused Variables
+
+A good practice is naming variables appropriately; however, variables may be unused at times such as the `timestamp` from the `call()`.
+
+```sway
+{{#include ../../../code/language/style-guide/unused_variables/src/lib.sw:named_unused_variable}}
+```
+
+When a variable is unused we can discard its value by assigning it to `_`.
```

Comment:
This should mention you can prefix with _ and conserve the naming for context, here as `_timestamp`.

Reply:
I have considered that and based on your comment in #4655 I've opted to exclude the name. I'll adjust it in a bit